src/rmc-compiler/rustc_codegen_rmc/src/utils/utils.rs

```diff
@@ -53,6 +55,10 @@ impl<'tcx> GotocCtx<'tcx> {
 
         Expr::statement_expression(body, t).with_location(loc)
     }
+
+    pub fn unsupported_msg() -> &'static str {
```

Comment:
Not a blocker, but I was expecting a function that gets the name of the function and generates the error message.